### ISSUE TITLE
goswagger server provides the API list in the debugging or explore mode

### ISSUE THREAD

**Issue by rzhang2403** [go-swagger/go-swagger#1210]
Based on the swagger 2.0 spec, I can quickly generate the swagger server codes and start to run the RESTful API server.  For the development, it could be nice to have a debugging mode (after enabled), it can use a browser to navigate the APIs and provide the UI to try the interface.

I used to work on the NodeJS framework,  Strongloop  loopback, which does a very good job in the API development support.   It can generate the API swagger 2.0 speci file, and create the "explore" page to developers.

Thanks.

**Comment by gregmarr** [comment]
Have you looked at the automatically generated `/docs` interface?  It does do execution, but has full documentation.

**Comment by rzhang2403** [comment]
You mean the file,  restapi/doc.go?
I found restapi/embedded_spec.go, which has the API spec.  But, this is not what I meant here.   Please check the link,

https://loopback.io/doc/en/lb3/Use-API-Explorer.html

**Comment by gregmarr** [comment]
No, I mean run the server and go to `http[s]://<server>/docs` in a browser.

**Comment by rzhang2403** [comment]
I just tried, and got:

{"code":404,"message":"path /docs was not found"}

**Comment by rzhang2403** [comment]
I saw that if I used the example, to-do-list, the /docs works to bring the nice UI.   My swagger 2.0 was created and validated in the go-swagger.   I don't know why it's broken here.

**Comment by rzhang2403** [comment]
I found that the problem is in the go-swagger doesn't support the spec, 
        basePath:  "/api"

After removed it, it works.

**Comment by gregmarr** [comment]
If you use a base path, then the docs comes after the base path.

**Comment by rzhang2403** [comment]
Yes, it works.  Thanks.
